webpack/assets/javascripts/react_app/components/CommonParameterForm/components/HiddenValueField.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { translate as __ } from '../../../common/I18n';
+import CommonForm from '../../common/forms/CommonForm';
+
+const HiddenValueField = ({ value, onChange }) => (
+  <CommonForm label={__('Hidden Value')}>
```

Comment:
Could we use `FormField` instead? IMHO `CommonForm` should get obsolated :thinking: